### PR TITLE
Mark SECURITY-1002 as fixed

### PR DESCRIPTION
**Pull request by reda-alaoui**
IMO, the security issue has been fixed starting with https://github.com/jenkinsci/sonar-gerrit-plugin/releases/tag/378.vf4646d4df087 .

The security issue was declared in https://www.jenkins.io/security/advisory/2022-12-07/#SECURITY-1002.

The regex was tested [here](https://regex101.com/r/TSGqk9/1).

### CODE REVIEW

**Review by daniel-beck — CHANGES_REQUESTED** [review]

File: resources/warnings.json [hunk 1]
```diff
@@ -14795,7 +14795,7 @@
     "versions": [
       {
         "lastVersion": "377.v8f3808963dc5",
-        "pattern": ".*"
+        "pattern": "([1-2]|3[0-7][0-7])\\..*"
```
[the remark this answers is not in the document]
Comment (reda-alaoui) [reply resources/warnings.json]: Indeed :man_facepalming:

**Comment by reda-alaoui** [comment]
Hello? :)

**Comment by Kevin-CB** [comment]
I was able to confirm locally that the vulnerability is fixed by https://github.com/jenkinsci/sonar-gerrit-plugin/commit/f4646d4df0870bd06b76de86309bb38fbd929d54 which got released in v378.vf4646d4df087

**Comment by Kevin-CB** [comment]
Once this PR is merged, the warning should be removed within a day.

**Comment by reda-alaoui** [comment]
Thank you !

**Pull request merged**